jscomp/build_tests/cli_help/input.js

```diff
@@ -32,10 +32,11 @@ const buildHelp =
   "`rescript build -- -h` for Ninja options (internal usage only; unstable)\n" +
   "\n" +
   "Options:\n" +
-  "  -w          Watch mode\n" +
-  "  -ws         [host]:port set up host & port for WebSocket build notifications\n" +
-  "  -verbose    Set the output to be verbose\n" +
-  "  -with-deps  *deprecated* This is the default behavior now. This option will be removed in a future release\n";
+  "  -w           Watch mode\n" +
+  "  -ws          [host]:port set up host & port for WebSocket build notifications\n" +
+  "  -verbose     Set the output to be verbose\n" +
+  "  -with-deps   *deprecated* This is the default behavior now. This option will be removed in a future release\n" +
+  "  -warn-error  Enable warnings as error\n";
```

Comment:
It is unclear here that a list of warning codes is needed as an argument (and what format this list needs to have).